pkgs/top-level/python-packages.nix

```diff
@@ -3504,9 +3504,9 @@ in {
 
   kmapper = callPackage ../development/python-modules/kmapper { };
 
-  kmsxx = toPythonModule ((callPackage ../development/libraries/kmsxx {
-    inherit (pkgs.kmsxx) stdenv;
-    inherit (pkgs) pkg-config;
+  kmsxx = toPythonModule ((pkgs.callPackage ../development/libraries/kmsxx {
```

Comment:
I am not sure if this mixes python versions. @jonringer

Reply:
I checked with `ldd` and in a python shell -- it's importable and doesn't look like there's evidence in ldd of multiple pythons or anything.